src/components/viz/ReviewCase.vue

```diff
@@ -628,10 +629,9 @@ export default {
       this.modelInfosData = this.modelInfos;
 
       this.setPedigree(this.pedigree); //Set state in Vuex store
-      
+
       this.populateRelationshipMap();
       this.populateSampleIdsFromCustom(this.pedigree);
-      this.populateSampleIdsAndRelationships();
```

Comment:
Hi @WilliamRichards2017 , 
`this.populateSampleIdsAndRelationships();`
Sorry for getting back to this PR. I just wanted to check why this line of code was removed? I noticed it was breaking the review-case page with custom data.

Reply:
That line should not be removed! That's super weird.  I added an extra commit after I requested a review to add back in that accidentally removed line.  When I check my file, I see that line was added back in.  I wonder if I accidentally pushed that commit adding the line back into the branch to fix the custom-data variant badge.  Sorry for this
<img width="1349" alt="Screen Shot 2020-06-26 at 3 30 49 PM" src="https://user-images.githubusercontent.com/15776714/85903118-ced7a980-b7c2-11ea-945a-9fa25c9aaf3d.png">

Reply:
Oh Ok! Np... I added that line back. It's working fine now!